### PR TITLE
modifie integration field nationality

### PR DESCRIPTION
**Pull request by Raulnet**
modifie integration field nationality

### CODE REVIEW

File: front/style/pages/_donation.scss [hunk 1]
```diff
@@ -389,6 +389,18 @@
         &__city {
             width: 208px;
         }
+        .form__row--stretch {
+            margin-bottom: 15px;
+            label.form__label {
+                padding: 20px 0;
+                font-size: 16px;
+                font-family: "Roboto", Helvetica, sans-serif;
```
Comment (thomasbisignani) [inline front/style/pages/_donation.scss]: You can use : `$font-roboto` here

File: front/style/pages/_donation.scss [hunk 2]
```diff
@@ -389,6 +389,18 @@
         &__city {
             width: 208px;
         }
+        .form__row--stretch {
+            margin-bottom: 15px;
```
Comment (thomasbisignani) [inline front/style/pages/_donation.scss]: You can use directly the `b__nudge--bottom-medium` class in your template for this margin bottom

File: templates/donation/informations.html.twig [hunk 3]
```diff
@@ -55,16 +55,17 @@
...
             <div class="form__row">
                 {{ form_errors(form.emailAddress) }}
                 {{ form_widget(form.emailAddress, { attr: { class: 'form--full', placeholder: 'Adresse email' } }) }}
             </div>
 
-            <div class="form__row">
-                {{ form_errors(form.nationality) }}
-                {{ form_widget(form.nationality, { attr: { class: 'form--full', placeholder: 'Nationalité' } }) }}
-            </div>
+
```
Comment (JeremieSamson) [inline templates/donation/informations.html.twig]: remove this line plz ;)

Comment (Raulnet) [reply templates/donation/informations.html.twig]: fixed

File: front/style/pages/_donation.scss (hunk 1 above)
Comment (Raulnet) [reply front/style/pages/_donation.scss]: fixed

File: front/style/pages/_donation.scss (hunk 2 above)
Comment (Raulnet) [reply front/style/pages/_donation.scss]: fixed

**Pull request closed without merging**